benchmarks/annotation_store.ipynb

```diff
@@ -0,0 +1,2236 @@
+{
```

Comment:
<p>In the toolbox, we ...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TissueImageAnalytics/tiatoolbox/pull/320/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>